modules/EmbedPlayer/resources/mw.EmbedPlayer.js

```diff
@@ -1321,6 +1319,9 @@
 			$(this).trigger('playerReady');
 			this.triggerWidgetLoaded();
 
+			// Update layout
+			this.doUpdateLayout();
+
```

Comment:
@GuyLivni @itaykinnrot  - updateLayout must come before we can trigger playerReady. We expect the DOM to be ready when the playerReady event is triggered. This change caused a regression: FEC-3389. Please advise.